### DISCUSSION TITLE
Looking for final code changes for syncing events

### DISCUSSION THREAD

**Discussion by dribnus** [derekantrican/GAS-ICS-Sync#335] [General]
I've seen a lot of good info for editing current code to allow past events to not be deleted, yet delete upcoming events if they are deleted from the source before they occur.  My issue is I've found too much info.  LOL, it has become overwhelming.  Is there anyway to get the final code changes that are needed for the most recent script that came out in Jan?

**Comment by jonas0b1011001** [comment]
The feature officially made to to the master branch. You can manually copy [_code.gs_](https://github.com/derekantrican/GAS-ICS-Sync/blob/master/Code.gs) and [_helpers.gs_](https://github.com/derekantrican/GAS-ICS-Sync/blob/master/Helpers.gs) from github and use [_var removePastEventsFromCalendar_](https://github.com/derekantrican/GAS-ICS-Sync/blob/e5cd8e02417ee38a04eadbd795201822dbac2e2d/Code.gs#L39).

**Reply by dribnus** [reply]
Perfect.  Thanks so much for the quick response too.  Time to donate.  This script is amazing and I know took a lot of work.  Thank you.
